javatests/arcs/core/host/lifecycle.arcs

```diff
@@ -67,3 +67,32 @@ recipe ReadWriteAccessTest
     colReadWrite: reads writes h2
     sngPersist: reads writes h3
     colPersist: reads writes h4
+
+// -----------------------------------------------------------------------------
+
+particle PipelineProducerParticle in 'arcs.core.host.PipelineProducerParticle'
```

Comment:
Since the namespace is `arcs.core.host`, this particle location need only be `'.PipelineProducerParticle'`. Same for the particles below.

Reply:
Done